doc/reference/configuration/cfg_replication.rst

```diff
@@ -32,9 +32,13 @@
 
     :extsamp:`box.cfg{ replication = { {*{'uri1'}*}, {*{'uri2'}*} } }`
 
-    If one of the URIs is "self" -- that is, if one of the URIs is for the
-    instance where ``box.cfg{}`` is being executed on -- then it is ignored.
-    Thus it is possible to use the same ``replication`` specification on
+    ..  note::
+
+        Starting from version 2.10.0, there is a number of other ways for specifying several URIs. See :ref:`syntax examples <index-uri-several>`.
+
+    If one of the URIs is "self"---that is, if one of the URIs is for the
```

Comment:
I don't really understand this phrase

Reply:
Can be fixed in a separate ticket because it's from some old descritption.